cloud/docker/docker.py

```diff
@@ -1322,7 +1323,10 @@ def do_create(count, params):
 
         try:
             containers = do_create(count, params)
-        except:
+        except docker.errors.APIError as e:
```

Comment:
not 2.4 compatible, but the module already requires 2.6 cause of docker.py, so not an issue.